Schedulers contract update after security review (#669)
rsksmart/devportal: rif/scheduler/run.md

You will find one for local networks,
other for RSK Testnet and other for RSK Mainnet.

> *WARNING: Please notice that the contract requires Solidity 0.8 or newer. Compiling it with older versiosn will expose it to security vulerabilities.*   

Reviewer: ```suggestion
> WARNING: Please note that the contract requires Solidity 0.8 or newer. Compiling it with older versions will expose it to security vulnerabilities.
```

In the deployment you will need to provide two addresses in the constructor:

- `address serviceProvider_`:
  Can create and remove plans. Can change the payee.
- `address payee_`:
  Receives the tokens paid after executions.
- `uint256 minimumTimeBeforeExecution_`:

>   Minimum amount of seconds required between the moment of the scheduling and the execution time requested.
>   This should be grater then 15 and protects the service provider from timestamps manipulation.

Reviewer: ```suggestion
  Minimum amount of seconds required between the moment of the scheduling and the execution time requested.
  This should be greater than 15, and protects the service provider from manipulation of timestamps.
```

>   This should be greater than 15, and protects the service provider from manipulation of timestamps.

Reviewer 2: ```suggestion
  This must be greater than 15, it protects the service provider from timestamp manipulation.
```
